flight/Modules/Stabilization/stabilization.c

```diff
@@ -105,6 +105,15 @@ enum {
 	PID_MAX
 };
 
+typedef struct {
```

Comment:
In general we don't do typedefs like this, except for things we opaquify and export from modules as handles.  On this one I could go either way..